.gitignore

```diff
@@ -1,2 +1,5 @@
 /workflows/generated-files/
 /workflows/__pycache__/
+/workflows/dashboard_matrix/venv/
```

Comment:
please add also /workflows/dashboard_martix/ouput/

Reply:
you can also use a more generic pattern for `venv`, i.e. `venv/` because I don't believe we want to commit these no matter where in the repo they're located. Same with `__pycache__/` please